pages/contributors.js

```diff
@@ -0,0 +1,220 @@
+                    </Text>
+                    <Text tag="h2" textSize="title" textColor="#003e54">
+                        Build products, practice skills, learn technologies, win
+                        prizes,and connect with people.
+                    </Text>
+                </div>
+            </div>
+            {error[0] ? (
+                <div className="text-center">
+                    Unable to fetch data, please try again!
+                </div>
+            ) : (
+                <>
+                    {contributors && contributors.length ? (
+                        <div className="container py-4">
+                            <div className="row no-gutters mt-2">
+                                {contributors.map((item, index) => (
+                                    <div
+                                        className="col-12 col-md-6 col-lg-3 p-3"
+                                        key={item.handle_name}
+                                    >
+                                        <Card
+                                            name={item.name}
+                                            image={item.image}
+                                            github={item.github}
+                                            description={item.description}
+                                        ></Card>
+                                    </div>
+                                ))}
+                            </div>
+                        </div>
+                    ) : contributors ? (
+                        <div className="text-center">
+                            <Lottie options={defaultOptions} height={300} />
+                            <Text
+                                tag="h6"
+                                textSize="subheader"
+                                textColor="003e54bd"
+                                fontFamily="madetommy-bold"
+                            >
+                                No Contributors Right Now
+                            </Text>
+                        </div>
+                    ) : (
+                        <div className="text-center">
+                            <Spinner animation="border" role="status">
+                                <span className="sr-only">Loading...</span>
+                            </Spinner>
+                        </div>
+                    )}
+                </>
+            )}
+            {/* <div className="container py-4">
+                <div className="row no-gutters mt-2">
+                    {contributors.map((item, index) => (
+                        <div className="col-12 col-md-6 col-lg-3 p-3" key={item.handle_name}>
+                            <Card name={item.name} image={item.image} github={item.github}>
+                                {item.description}
+                            </Card>
+                        </div>
+                    ))}
+                </div>
+            </div> */}
```

Comment:
If of no use, you can remove this block of code

Reply:
I ll remove this now